Dockerfile

```diff
@@ -40,12 +39,19 @@ RUN yum install -y git \
 
 
 # various system deps
-RUN yum install -y epel-release \
- && yum install -y redis \
+RUN echo $'\n\
+[google-chrome]\n\
+name=google-chrome\n\
+baseurl=http://dl.google.com/linux/chrome/rpm/stable/x86_64\n\
+enabled=1\n\
+gpgcheck=1\n\
+gpgkey=https://dl-ssl.google.com/linux/linux_signing_key.pub' \
+ > /etc/yum.repos.d/google-chrome.repo \
+  && yum install -y epel-release \
+  && yum install -y redis \
                    mysql-devel \
-                   Xvfb \
                    firefox \
-                   qt5-qtwebkit-devel \
+                   google-chrome-stable \
                    libicu \
```

Comment:
@hallelujah I wonder... is this a qt requirement? Could we get rid of it too? 

Or do we need it for something else?

Reply:
The last commit does not have it

Reply:
Ah sorry, I misread the line (too late here)
I do not know :)
Probably yes I will try removing it tomorrow

For now the image is hosted at quay.io/3scale/system-builder:headless

Reply:
@hallelujah what about `libicu` then ? any idea if it could also be removed?

Reply:
btw - am I remembering correctly that we no longer need qt at all !?

Reply:
We do not need qt at all
Recent tests with QT took 9' on cucumbers and without it but plain browser took 7'
libicu I will try to do a build without it and see if it was a qt dependency

Reply:
nice - let us know how it goes ;)